app/frontend/src/components/ProjectItem/type.ts

```diff
@@ -0,0 +1,5 @@
+export interface Repo {
+    id?: number
```

Comment:
is it because of the `?` next to `id` so that you do not need to use `;` at the end?

Reply:
The ´?´ makes the attribute become optional so you can choose not to use it, but I will do a little change and add the ´;´ at the end because that is the good code standard